### ISSUE TITLE
Feat: Cache control with cloudflare

### ISSUE THREAD

**Issue by DarkPhoenix2704** [Dokploy/dokploy#232]
### What problem will this feature address?

Some of my deployed frontend bundles are very large and has to be cached using the cloudflare. But everytime, I update the container, I have to manually clear the cache from the cloudflare

### Describe the solution you'd like

It would be nice to have an wrapper around cloudflare cache using their apis. We can configure the caches by paths, headers, etc and revalidate the caches on successfull builds

### Describe alternatives you've considered

Manage the cache from cloudflare dashboard

**Comment by Siumauricio** [comment]
Hi @DarkPhoenix2704, I would like to integrate more things, but this is something very specific to cloudflare and I don't like it too much, isn't there a way that you can integrate this through the dokploy API in some way to automate this?

**Comment by yayza** [comment]
I don't think it's really needed on Dokploy. I was researching Cloudflare's API based on your post and came across [this gist](https://gist.github.com/vdbelt/20f116236d2ebffa92f131e679c0551a), which seems really easy to implement.

TLDR: Copy/paste the script to cloudflare worker, [point a route to it in Cloudflare dashboard](https://developers.cloudflare.com/workers/configuration/routing/routes/), and bookmark it to your browser for easy purging.
